project/profiles/capacity4more/modules/c4m/user/c4m_user_subscribe/c4m_user_subscribe.module

```diff
@@ -106,3 +106,24 @@ function _c4m_user_subscribe_entity_notifications_disabled($entity_id, $flag_suf
 
   return FALSE;
 }
+
+/**
+ * Implements hook_og_membership_type_insert().
+ *
+ * Flags the group subscription flag, so the user is subscribed to the group.
+ */
+function c4m_user_subscribe_og_membership_insert(OgMembership $membership) {
+  $flag = flag_get_flag('subscribe_c4m_follow_group');
+  $flag->flag('flag', $membership->gid, NULL, TRUE);
```

Comment:
ok, for sure, it is clearly written in the task.

Only when the membership is active